src/patch-builtins.js

```diff
@@ -333,6 +333,15 @@ export function patchBuiltins() {
     if (window.HTMLSlotElement) {
       patchAccessors(window.HTMLSlotElement.prototype);
     }
+    // See https://html.spec.whatwg.org/multipage/forms.html#form-associated-element
```

Comment:
This won't work on browser on which we cannot patch prototypes (Safari 9 and older Chrome). This may be ok but it at least deserves a comment.